client/app/components/Modal/index.jsx

```diff
@@ -1,10 +1,9 @@
 // @flow
 import React from 'react';
 import renderHTML from 'react-render-html';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
We'll also want to replace the close icon in the Header component as well to be consistent. You can use CSS to change the fill colour of it.

Reply:
Reverted the SVG back to FAIcons as per discussion below.